jobs/haproxy/templates/haproxy.config.erb

```diff
@@ -169,6 +169,9 @@ end
 # }}}
 
 # Error checking
+  if !p("ha_proxy.accept_proxy", true) && !p("ha_proxy.expect_proxy", nil)
```

Comment:
there should be a spect test for this in the haproxy.config frontend tests, to ensure that the flags are mutually exclusive.

Reply:
Thanks, will do.

Reply:
done